sentry-sample/src/main/cpp/native-sample.cpp

```diff
@@ -14,10 +14,24 @@ JNIEXPORT void JNICALL Java_io_sentry_sample_NativeSample_crash(JNIEnv *env, jcl
 
 JNIEXPORT void JNICALL Java_io_sentry_sample_NativeSample_message(JNIEnv *env, jclass cls) {
     __android_log_print(ANDROID_LOG_WARN, TAG, "Sending message.");
+
+    jclass clazz = env->FindClass("io/sentry/sample/NativeSample");
+    jmethodID method = env->GetStaticMethodID(clazz, "test",
+                                              "()Ljava/lang/String;");
+    jstring result = (jstring) env->CallStaticObjectMethod(clazz, method);
+    // if I do a NPE check, it bails out and it works
+//    if (result == NULL) {
+//        return;
```

Comment:
Let's try to see what happens with the uncaught exception handler here if we do `env->ExceptionClear(env)` before the return.